pkgs/by-name/st/stu/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stu,
+  testers,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Comment:
Consider removing the `rec` for the `let in` synatx, see here https://nix.dev/guides/best-practices.html.